config/deploy/prod.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
-server 'sul-robots1-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app db}
-server 'sul-robots2-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app}
-server 'sul-robots3-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app}
-server 'sul-robots4-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app}
-server 'sul-robots5-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app}
+server 'common-accessioning-prod-a.stanford.edu', user: 'lyberadmin', roles: %w{web app db} # only server to get whenever cronjobs deployed to
```

Comment:
should there be a cap "whenever" role to make this explicit?

Reply:
Whenever has to happen on all of the servers, but one whenever thing only happens on one server.  This hasn't changed from how we currently have it.